lib/src/widgets/pgn.dart

```diff
@@ -177,14 +185,20 @@ class _DebouncedPgnTreeViewState extends ConsumerState<DebouncedPgnTreeView> {
   void didUpdateWidget(covariant DebouncedPgnTreeView oldWidget) {
     super.didUpdateWidget(oldWidget);
 
-    if (oldWidget.currentPath != widget.currentPath) {
-      // debouncing the current path change to avoid rebuilding when using
-      // the fast replay buttons
+    if (oldWidget.currentPath != widget.currentPath ||
+        oldWidget.broadcastLivePath != widget.broadcastLivePath) {
+      // debouncing the current and broadcast live path changes to avoid rebuilding when using
+      // the fast replay buttons or when receiving a lot of broadcast moves in a short time
       _debounce(() {
         setState(() {
-          pathToCurrentMove = widget.currentPath;
+          if (oldWidget.currentPath != widget.currentPath) {
+            pathToCurrentMove = widget.currentPath;
+          }
+          if (oldWidget.broadcastLivePath != widget.broadcastLivePath) {
+            pathToBroadcastLiveMove = widget.broadcastLivePath;
+          }
         });
-        WidgetsBinding.instance.addPostFrameCallback((_) {
+        if (oldWidget.currentPath != widget.currentPath) {
```

Comment:
Should should not remove the postFrameCallback here. This change looks like a mistake.

Reply:
Completely a mistake. Good catch